runner/src/elf2tab.rs

```diff
@@ -37,6 +63,7 @@ pub fn convert_elf(cli: &Cli) -> OutFiles {
     }
 
     let mut command = Command::new("elf2tab");
+
```

Comment:
Did `rustfmt` insert this newline? It seems unnecessary.

Reply:
Not sure